src/main/java/com/example/demo/repository/TeamsList.java

```diff
@@ -0,0 +1,61 @@
+package com.example.demo.repository;
+
+import com.example.demo.entity.Team;
+import com.example.demo.service.SQLMethods;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class TeamsList {
+    List<Team> teams = new ArrayList<>();
+    SQLMethods sqlMethods;
+
+    Team Monday = new Team("Monday", "M", "N");
```

Comment:
Stick to [Java Code Conventions](https://www.oracle.com/technetwork/java/codeconventions-150003.pdf).